### DISCUSSION TITLE
Is there a way to limit request body size?

### DISCUSSION THREAD

**Discussion by JJPell** [ThreeMammals/Ocelot#2192] [Q&A] [answered]
Hi,

Is there a way to either limit or configure the maximum request body size?

Thanks

**Comment by raman-m** [comment, marked as answer]
Read "**Maximum request body size**" of the [Ocelot | Gotchas | Kestrel](https://ocelot.readthedocs.io/en/latest/introduction/gotchas.html#kestrel) docs.
- MS Learn: [Configure options for the ASP.NET Core Kestrel web server | Maximum request body size](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/options?view=aspnetcore-8.0#maximum-request-body-size)
